app/components/footer_component.html.erb

```diff
@@ -66,42 +60,34 @@
           </ul>
         </div>
       </div>
-
       <hr class="site-footer__divider">
     </section>
-
     <div class="site-footer__social-icons">
       <h2 class="visually-hidden">Support links</h2>
-      <a href="https://www.facebook.com/getintoteaching" class="site-footer-top__social__link" target="_blank" rel="noopener">
+      <a href="https://www.facebook.com/getintoteaching" class="site-footer-top__social__link">
         <%= helpers.fab_icon("facebook-f", "icon") %>
-        <span class="visually-hidden">Facebook - opens in a new tab</span>
```

Comment:
@sarahcrack I think we need to keep the visually hidden line for each of the social media icons with the social media site name, but without the `opens in new tab`

If we don't have this visually hidden information, there is no way for screen reader users to understand what these links are

ie

*  `<span class="visually-hidden">Facebook</span>`
*  `<span class="visually-hidden">Instagram</span>`
*  `<span class="visually-hidden">LinkedIn</span>`
*  `<span class="visually-hidden">X, formerly known as Twitter</span>`
*  `<span class="visually-hidden">YouTube</span>`

Reply:
thanks @gemmadallmandfe! I am going to test with aria labels initially as I think this should be sufficient for screen readers if the links are opening in the same tab. I will add/test with visually-hidden text if aria labels don't work as expected. 

Facebook example with aria label:
` <a href="https://www.facebook.com/getintoteaching" class="site-footer-top__social__link" aria-label="Facebook"><%= helpers.fab_icon("facebook-f", "icon") %></a>`

Reply:
Tested successfully using iOS/VoiceOver :-)